scheduler/src/cook/pool.clj

```diff
@@ -16,14 +16,25 @@
 
 (def nil-pool "nil-pool")
 
+(defn check-pool-and-submit-pool
```

Comment:
This replaces `check-pool` (ln 6) for the `/jobs` endpoint. The other function is also used for quota and share, so I did not want to touch those.

Reply:
I think it might be worth adding a similar comment in the code; eg for the other function, say that it  checks for pool specifically so it can be used for quota and share; for this function, say that it should be used for logic that checks for job submitted to the pool